docs/cli-usage.md

```diff
@@ -0,0 +1,96 @@
+## Using Kubernetes from the command line
```

Comment:
I don't necessarily want to merge this, but I believe that this is the PR represents the min-bar for a usable client experience.  I intend it to be a roadmap for the development work I'm going to begin ~now.  However, I wanted to have the discussion before I do the development.